.changeset/tame-buckets-move.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure-devops': patch
+'@backstage/plugin-azure-devops-backend': patch
+'@backstage/plugin-azure-devops-common': patch
```

Comment:
From what I can see I think that this is a `minor` change for the `common` package

Reply:
Sounds good, bit of a learning experience for me. I've made this change, thanks 👍